docs/real-world-examples/ccfraud.md

```diff
@@ -31,9 +31,33 @@ In the next section, we will run a job in the AzureML workspace that will unpack
 
 Kaggle required username and a developer key, so we will first store safely those credentials in the workspace keyvault.
 
+### Option 1: using Azure CLI
+
+1. In this section, we'll create an access policy for yourself, by using your AAD identifier (object id). 
```

Comment:
This sentence is a bit confusing. Maybe something like: "1. Let's first obtain your AAD identifier (object id) by running the following command. We'll use it in the next step."